### ISSUE TITLE
Create an image slideshow widget

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#63]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [x] Content editor
- [ ] Marketer
- [ ] Administrator
- [ ] Developer

### Needs what

To insert image slideshows.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`

**Comment by JanLenoch** [comment]
Resolved via acef058bad26663261d4b6aed3591cbc1787764b
